docs/installation.rst

```diff
@@ -0,0 +1,14 @@
+.. _installation:
+
+Installation
+============
+
+To install skops, run the following command in your Python environment:
+
+.. code-block:: bash
+
+    python -m pip install skops
```

Comment:
I'm not sure why you prefer this to `pip install skops`.

Reply:
This the recommended way in [the docs](https://docs.python.org/3/installing/index.html). The reason I've seen in the past is that even though in general it's the same, in some situations `pip install` can lead to the package being installed globally, instead of the specific python environment. In contrast, `python -m pip install` will never fail where `pip install` works.